templates/matlab.yml

```
@@ -0,0 +1,43 @@
+# MATLAB
+# Run MATLAB and Simulink as part of your build pipeline.
+# Add steps that install MATLAB, filetr tests run, generate and publish artifacts, deploy, and more:
+# https://marketplace.visualstudio.com/items?itemName=MathWorks.matlab-azure-devops-extension
+
+trigger:
+- {{ branch }}
+
+pool:
+  {{ pool }}
+
+steps:
+# Uncomment this task to install MATLAB on a Microsoft-hosted agent.
+#- task: InstallMATLAB@0
+#  inputs:
+#    release: latest
+
+# Uncomment this task to run a specific MATLAB command or script.
+#- task: RunMATLABCommand@0
+#  inputs:
+#    command: myscript
+
+
+- task: RunMATLABTests@0
+  inputs:
+    sourceFolder: mySrc;mySrc2
+    testResultsPDF: test-results/results.pdf
+    testResultsJUnit: test-results/results.xml
+    codeCoverageCobertura: code-coverage/coverage.xml
+
+
+# Tasks that help publish the test artifacts generated
+- task: PublishBuildArtifacts@1
+  inputs:
+    pathToPublish: test-results/results.pdf
+- task: PublishTestResults@2
+  condition: succeededOrFailed()
+  inputs:
+    testResultsFiles: test-results/results.xml
+- task: PublishCodeCoverageResults@1
+  inputs:
+    codeCoverageTool: Cobertura
+    summaryFileLocation: code-coverage/coverage.xml
```

I think we can simplify.

```suggestion
- task: RunMATLABTests@0
  inputs:
    testResultsJUnit: test-results.xml

- task: PublishTestResults@2
  condition: succeededOrFailed()
  inputs:
    testResultsFiles: test-results.xml
```

---

done. Looks pretty simple than proposed design though :)